iXBRLViewerPlugin/viewer/src/js/textblockviewer.js

```diff
@@ -0,0 +1,47 @@
+// Copyright 2023 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Dialog } from './dialog.js';
+
+export class TextBlockViewerDialog extends Dialog {
+    constructor() {
+        super(".dialog.text-block-viewer");
```

Comment:
(Arbitrary line.)

What were your thoughts on adding some kind of checkbox to switch between the "basic" text-only view from before and this rendered view?

Reply:
CSS style that interferes with "good" rendering.  For example, Workiva historically produced inline styles that caused absolute positioning of elements relative to a page `<div>`, so if continuations crossed multiple pages, then all `<div>`s can overlap and become unreadable if you don't ignore the styling as ixbrl-viewer does today.  Currently, that is a non-issue because the formatting is mostly removed, so it might be nice to give a toggle for that mostly-plain-text view that existed previously.  There might be other use cases too.

Reply:
I've had a go at this in 8e7ba9e

![image](https://user-images.githubusercontent.com/45077928/224182923-771ddaf7-29d1-459d-9bff-1bb5bd6a0c96.png)